Document_in_Progress.md

```diff
@@ -0,0 +1,87 @@
+
+In May 2018 the NZSL Android app got up to the number 6 ranking in the worldwide category “Top Free in Books and Reference” apps on Google Play.
+
+New Zealand Sign Language Dictionary consists of 3 major units
+* Ruby on Rails Website (this repo)
+* Mobile apps <add address>
+* Freelex, (add address) an open source project for maintaining the signs lexicon data.
+
+[![Build Status](https://travis-ci.org/ODNZSL/nzsl-online.svg?branch=master)](https://travis-ci.org/ODNZSL/nzsl-online)
+[![Code Climate](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/gpa.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online)
+[![Test Coverage](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/coverage.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online/coverage)
+[![Issue Count](https://codeclimate.com/github/ODNZSL/nzsl-online/badges/issue_count.svg)](https://codeclimate.com/github/ODNZSL/nzsl-online)
+
+## The Set Up for you wonderful open source contributors
+
+Please read carefully the CONTRIBUTORS.md (add link?) before you follow the setup steps.
+
+Fork the official repo [ODNZSL repo](https://github.com/ODNZSL/nzsl-online), so you have a copy.
+Clone your copy
+
+```
+git clone <your copy>
+cp env-example .env
+bundle
+yarn
+bundle exec rails s
+```
+A suggestion is to create a local staging branch ( You can really call it what you want ) that acts as your local master. Then branch from your staging/master branch when resolving issues or adding features.
+
+  * Create your feature branch. For example `NZSL-##Bugfix/Back-to-search-results`
+  * Make changes
+  * Commit you changes
+  * Push to the base staging branch
+  * Create a new pull request
+
+## Issues (user stories) that require design and a design process
+
+a high level design story outcome from design team
+
+## Brand guidelines
+
+## Tech requirements ??
+
+Ruby on Rails, HAML, BEM
+
+## Release History ??
+
+## Meta ??
+
+## Deployment
+
+NZSL has taken ownership of the project at their [ODNZSL repo](https://github.com/ODNZSL/nzsl-online).
+
+Open source contributors pull requests should be to the ODNZSL Staging branch. The admin for ODNZSL is currently [Elspeth Dick](elspeth@rabidtech.co.nz).
+
+Open source contributors should merge the ODNZSL staging branch to your local staging branch so that your repo is up-to-date.
+
+
+To deploy to production: merge ONZSL staging to ODNZSL master.
+
+## Environments:
+
+Current servers (2017-12-15)
```

Comment:
Staging is behind basic auth right now. I have no real reckons on this but what should be the process if contributors have their changes deployed to staging but then have their work rejected because of bugs that they cant replicate locally?

Reply:
i think it's okay to document the user/pass - the password is on staging mostly to stop a member of the public going there by accident, and to ensure nothing links or indexes it as if it was the real thing.